cargo test error: "cc" did not execute successfully

Just a fyi...

$ sudo apt-get install pkg-config libssl-dev
[sudo] password for maswanso:
Reading package lists... Done
Building dependency tree
Reading state information... Done
pkg-config is already the newest version.
libssl-dev is already the newest version.

$ cargo test
error: failed to run custom build command for `systest v0.1.0 (file:///home/user/src/rust/rust-openssl/systest)`
process didn't exit successfully: `/home/user/src/rust/rust-openssl/target/debug/build/systest-23feef26bd8524a2/build-script-build` (exit code: 101)
--- stdout
-----------------------------------------
cargo:rerun-if-changed=../openssl-sys/src/lib.rs
cargo:rerun-if-changed=../openssl-sys/src/openssl/mod.rs
cargo:rerun-if-changed=../openssl-sys/src/openssl/v10x.rs
OPT_LEVEL = Some("0")
HOST = Some("x86_64-unknown-linux-gnu")
HOST = Some("x86_64-unknown-linux-gnu")
CC_x86_64-unknown-linux-gnu = None
CC_x86_64_unknown_linux_gnu = None
HOST_CC = None
CC = None
HOST = Some("x86_64-unknown-linux-gnu")
HOST = Some("x86_64-unknown-linux-gnu")
CFLAGS_x86_64-unknown-linux-gnu = None
CFLAGS_x86_64_unknown_linux_gnu = None
HOST_CFLAGS = None
CFLAGS = None
DEBUG = Some("true")
running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-m64" "-Wall" "-Wextra" "-Wall" "-Wextra" "-Werror" "-Wno-unused-parameter" "-Wno-type-limits" "-o" "/home/user/src/rust/rust-openssl/target/debug/build/systest-60bc9c78bd8adfa3/out/all.o" "-c" "/home/user/src/rust/rust-openssl/target/debug/build/systest-60bc9c78bd8adfa3/out/all.c"
cargo:warning=/home/user/src/rust/rust-openssl/target/debug/build/systest-60bc9c78bd8adfa3/out/all.c:2848:66: error: ‘X509_V_ERR_UNSPECIFIED’ undeclared here (not in a function)
cargo:warning=             static int __test_const_X509_V_ERR_UNSPECIFIED_val = X509_V_ERR_UNSPECIFIED;
cargo:warning=                                                                  ^
exit code: 1

--- stderr
thread 'main' panicked at '

Internal error occurred: Command "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-m64" "-Wall" "-Wextra" "-Wall" "-Wextra" "-Werror" "-Wno-unused-parameter" "-Wno-type-limits" "-o" "/home/user/src/rust/rust-openssl/target/debug/build/systest-60bc9c78bd8adfa3/out/all.o" "-c" "/home/user/src/rust/rust-openssl/target/debug/build/systest-60bc9c78bd8adfa3/out/all.c" with args "cc" did not execute successfully (status code exit code: 1).

', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/cc-1.0.18/src/lib.rs:2181:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.

--- a/openssl-sys/build/cfgs.rs
+++ b/openssl-sys/build/cfgs.rs
@@ -25,6 +25,9 @@ pub fn get(openssl_version: Option<u64>, libressl_version: Option<u64>) -> Vec<&
         if openssl_version >= 0x1_00_02_00_0 {
             cfgs.push("ossl102");
         }
+        if openssl_version >= 0x1_00_02_06_0 {
+            cfgs.push("ossl102f");
+        }
         if openssl_version >= 0x1_00_02_08_0 {
             cfgs.push("ossl102h");
         }

--- a/openssl-sys/src/openssl/mod.rs
+++ b/openssl-sys/src/openssl/mod.rs
@@ -33,6 +33,7 @@ pub const SSL_OP_NO_DTLSv1: c_ulong = 0x04000000;
 #[cfg(ossl102)]
 pub const SSL_OP_NO_DTLSv1_2: c_ulong = 0x08000000;
 
+#[cfg(ossl102f)]
 pub const X509_V_ERR_UNSPECIFIED: c_int = 1;
 
 pub const CMS_TEXT: c_uint = 0x1;

